cypress/e2e/flow-builder/blocks/moving.cy.ts

```diff
@@ -0,0 +1,74 @@
+import {IBlock} from '@floip/flow-runner'
+import Chainable = Cypress.Chainable
+
+describe('Moving a block', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'Moving a block',
+    })
+
+    cy.addBlock(['Content', 'Message']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`)
+        .find('[data-cy^="block-handle--"]')
+        .as('blockArea')
+    })
+
+    // wait for flow validation to succeed and flow validation toolbar to disappear
+    // to avoid interference with block coordinates
```

Comment:
And this time we need to `wait` because the flow validation might end in the middle of block move and mess up the coordinates.